infra/resources/_modules/apim/api/ioweb/_base_policy.xml

```diff
@@ -0,0 +1,53 @@
+<policies>
+    <inbound>
+        <base />
+        <cors>
+            <allowed-origins>
+                <origin>https://ioapp.it/</origin>
+            </allowed-origins>
+            <allowed-methods preflight-result-max-age="300">
+                <method>OPTIONS</method>
+                <method>GET</method>
+                <method>PUT</method>
+            </allowed-methods>
+            <allowed-headers>
+                <header>Content-Type</header>
+                <header>Authorization</header>
+            </allowed-headers>
+        </cors>
+        <!-- validate the jwt using the specified OpenID configuration URLs. Both URLs are used to ensure token validation across different identity providers -->
+        <validate-jwt header-name="Authorization" require-scheme="Bearer" output-token-variable-name="jwt">
+            <openid-config url="https://wallet.io.pagopa.it/hub-spid-login/openid-configuration" />
+            <openid-config url="https://wallet.io.pagopa.it/exchange/openid-configuration" />
+            <required-claims>
+                <claim name="fiscal_number" />
+            </required-claims>
+        </validate-jwt>
+        <!-- get the claim fiscal_number -->
+        <set-variable name="fiscalCode" value="@((string)((Jwt)context.Variables["jwt"]).Claims.GetValueOrDefault("fiscal_number"))" />
+        <!-- send the request to trial system -->
+        <send-request mode="new" response-variable-name="trialSystemResponse" timeout="5">
+            <set-url>@($"https://api.trial.pagopa.it/manage/api/v1/trials/{{trial-id}}/subscriptions/{context.Variables["fiscalCode"]}")</set-url>
```

Comment:
what if this call fails? myabe we should return some explicit error to the caller ie. "Error retrieving activation status"

Reply:
Added a TODO comment